UI.py

```diff
@@ -0,0 +1,109 @@
+from getpass import getpass
+# importing 'mysql.connector' as mysql 
+import mysql.connector as mysql
+
+class DbConnectionError(Exception):
+    pass
+
+#DB Connection
+def _connect_to_db():
+    cnx = mysql.connect(
+        host = "localhost",
+        user = "root",
+        passwd = "password",    #Reviewer Password
+        database = "walk2zero"
```

Comment:
After moving to db_utils.py, we will need to import variables from the config.py file `from config import USER, PASSWORD, HOST`, and change string values to those variables